includes/admin/importers/class-wc-product-csv-importer-controller.php

```diff
@@ -729,7 +733,7 @@ protected function get_mapping_options( $item = '' ) {
 			),
 			'reviews_allowed'    => __( 'Allow customer reviews?', 'woocommerce' ),
 			'purchase_note'      => __( 'Purchase note', 'woocommerce' ),
-			'meta:' . $meta      => __( 'Import as meta', 'woocommerce' ),
+			'meta:' . $meta      => __( 'Custom', 'woocommerce' ),
```

Comment:
We should still keep as "Import as meta" or "Import as meta data", custom don't means so much here.